storage/innobase/fts/fts0que.cc

```diff
@@ -953,6 +953,18 @@ fts_query_free_doc_ids(
 	query->total_size -= SIZEOF_RBT_CREATE;
 }
 
+/**
```

Comment:
InnoDB code (everything in `storage/innobase*`) requires InnoDB-style code formatting:
1. Tab characters must be used for indentation
2. Blocks must be indented by 1 tab
3. 1 tab = 8 spaces (set this in your editor)

SQL code (for instance `sql/*` and `include/*`) requires MySQL-style code formatting:
1. Using spaces only for indentation.
2. Blocks must be indented by 2 spaces.